packages/aws-cdk-lib/core/lib/private/synthesis.ts

```diff
@@ -254,6 +261,90 @@ function invokeAspects(root: IConstruct) {
   }
 }
 
+/**
+ * Invoke aspects V2 runs a stabilization loop and allows Aspects to invoke other Aspects.
+ * This function also does not emit a warning for nested Aspects. Runs if the feature flag
```

Comment:
Please find a different term than "nested aspects". "Nested" implies some sort of static structure to me that is not there.

Also -- do we ever get errors?